app/views/handlers/list_collector.py

```diff
@@ -38,3 +41,29 @@ def handle_post(self, form):
             self.questionnaire_store_updater.save()
         else:
             return super().handle_post(form)
+
+    def _render_block(self, block_id):
+        block_schema = self._schema.get_block(block_id)
+        transformed_block = transform_variants(
+            block_schema,
+            self._schema,
+            self._questionnaire_store.metadata,
+            self._questionnaire_store.answer_store,
+            self._questionnaire_store.list_store,
+        )
+
+        placeholder_renderer = PlaceholderRenderer(
+            language=self._language,
+            answer_store=self._questionnaire_store.answer_store,
+            metadata=self._questionnaire_store.metadata,
+        )
+
+        try:
+            summary = transformed_block.pop('summary')
+        except KeyError:
+            return placeholder_renderer.render(transformed_block)
+
+        return {
+            **placeholder_renderer.render(transformed_block),
+            **{'summary': summary},
+        }
```

Comment:
I think we can change the render block behaviour for all blocks to just render the question rather than the whole block. Then the list collector wouldn't need to override the render_block method.